packages/loot-core/src/server/budget/goaltemplates.ts

```diff
@@ -385,8 +374,8 @@ async function applyCategoryTemplate(
             ? template.repeat
             : (template.repeat || 1) * 12;
         while (num_months < 0 && repeat) {
-          target_month = addMonths(target_month, repeat);
-          num_months = differenceInCalendarMonths(
+          target_month = monthUtils.addMonths(target_month, repeat);
+          num_months = monthUtils.differenceInCalendarMonths(
             template_lines[l],
```

Comment:
I couldn't find a situation where this code is run, but we should be passig the month parameter instead of the whole template line.